### PR TITLE
Volume of Tetrahedron

### PR DESCRIPTION
**Pull request by pranjii**
### Describe your change:Added volume of tetrahedron

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by pandrey2003 — APPROVED** [review]
Had no idea before your PR we have different volume calculations in this repo 😄

File: maths/volume.py
Comment (pandrey2003) [inline maths/volume.py:236]: Err, forgot a small thing: put two spaces between code and comment, something like:
`print("Tetrahedron: " + str(vol_tetrahedron(2)))  # ~=0.94`
Currently, you have only 1 space which pylint does not like :)

**Comment by pranjii** [comment]
Gimme a minute

**Comment by pranjii** [comment]
@pandrey2003 is it ok now?

**Review by pandrey2003 — APPROVED** [review]
Looks stellar now!

**Comment by pandrey2003** [comment]
@pranjii Yep, awaiting a review from a developer with write access to the repo

**Comment by pranjii** [comment]
Thank u for helping me out

**Comment by cclauss** [comment]
In preparation for that denial-of-service attack that is [Hacktoberfest](https://hacktoberfest.com), I am closing all pull requests that do not have passing tests.

**Pull request closed without merging**
